nursery/capture-screenshot-go.yml

```diff
@@ -0,0 +1,31 @@
+rule:
+  meta:
+    name: capture screenshot (Go)
+    namespace: collection/screenshot
+    description: Detects screenshot capability via WinAPI for Go files.
+    author:
+      - joakim@intezer.com
+    scope: file
+    att&ck:
+      - Collection::Screen Capture [T1113]
+    mbc:
+      - Collection::Screen Capture::WinAPI [E1113.m01]
+  features:
+    - and:
+      - match: compiled with Go
```

Comment:
i think this is a great idea to limit the files the rule can match against.

Reply:
Yes, it needs to be limited some way to prevent potential false positives.